### DISCUSSION TITLE
questions about the engine

### DISCUSSION THREAD

**Discussion by realteam172** [KadeArchive/Kade-Engine#850] [Q&A] [answered]
is kade engine safe? can u do all note hold, trios, and doubles. im still trying to get help before installing this fnf engine. i want to play mods and its just questions about the engine. im a fnf pro. ill see u soon when u reply to me. goodbye for now. (awnsered).

**Comment by guizinhomartinez** [comment, marked as answer]
yes you can

**Reply by realteam172** [reply]
and is it safe to install

**Reply by ghost** [reply]
Yeah

**Reply by realteam172** [reply]
thanks. i can start playing the engine now. bye.

**Comment by Lucky-56** [comment]
please mark this as answered (top right of the answer that answered your question)
